benchmarks/benchmark_serving.py

```diff
@@ -163,14 +167,20 @@ def load_sharegpt_dataset(
   return dataset
 
 
-def load_openorca_dataset(dataset_path: str) -> list[tuple[Any, Any]]:
-  # Load the dataset.
-  with open(dataset_path, "r", encoding="utf-8") as f:
-    dataset = json.load(f)
+def load_openorca_dataset_pkl():
+  # read pickle file
+  samples = pandas.read_pickle(
+      os.path.join(
+          os.path.dirname(os.path.relpath(__file__)),
+          "open_orca_gpt4_tokenized_llama.calibration_1000.pkl",
```

Comment:
Lets pass the dataset path instead of hardcoding filename.

Reply:
the file has been checked into the same folder as the code itself.

Reply:
Yes but we may not be running the benchmarks only using this dataset as I commented above. So script should allow passing other files as well. Default can be this file.